### PR TITLE
Jusjohns xfcontrols switch

### PR DESCRIPTION
**Pull request by profexorgeek**
This PR adds a cross-platform sample project demonstrating the Xamarin.Forms Switch control.

### CODE REVIEW

**Comment by profexorgeek** [comment]
Forgot to add Readme file. Put do-not-merge tag on until I get a readme.md added.

File: UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/MainPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.ComponentModel;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace SwitchDemos
+{
+    // Learn more about making custom code visible in the Xamarin.Forms previewer
+    // by visiting https://aka.ms/xamarinforms-previewer
+    [DesignTimeVisible(false)]
```
Comment (davidbritch) [inline UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/MainPage.xaml.cs]: Remove this attribute (and comments). I know the templates add it, but it's not required for this page.

File: UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchBindingsPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace SwitchDemos
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
```
Comment (davidbritch) [inline UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchBindingsPage.xaml.cs]: Remove. Not required (already defined at the assembly level).

File: UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchCodePage.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            {
+                Text = "Switch state is OFF",
+                HorizontalOptions = LayoutOptions.CenterAndExpand,
+                VerticalOptions = LayoutOptions.CenterAndExpand
+            };
+
+            Switch switchControl = new Switch
+            {
+                IsToggled = false,
+                OnColor = Color.Green,
```
Comment (davidbritch) [inline UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchCodePage.cs]: Maybe go with a different colour, as it's pretty similar to the colour used on iOS already.

File: UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchXamlPage.xaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        <StackLayout>
+            <Label x:Name="SwitchStateLabel"
+                   Text="The switch is off"
+                   VerticalOptions="CenterAndExpand" 
+                   HorizontalOptions="CenterAndExpand" />
+
+            <Switch x:Name="XAMLSwitch"
+                    Toggled="XAMLSwitch_Toggled"
+                    IsToggled="False"
+                    OnColor="Green"
```
Comment (davidbritch) [inline UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchXamlPage.xaml]: Maybe go with a different colour, as it's pretty similar to the colour used on iOS already.

File: UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchXamlPage.xaml.cs [hunk 5]
```diff
@@ -0,0 +1,20 @@
+﻿using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace SwitchDemos
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
```
Comment (davidbritch) [inline UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchXamlPage.xaml.cs]: Remove. Already defined at assembly level.

File: UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchXamlPage.xaml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d"
+             x:Class="SwitchDemos.SwitchXamlPage"
+             Title="Switch XAML Demo">
+    <ContentPage.Content>
+        <StackLayout>
+            <Label x:Name="SwitchStateLabel"
```
Comment (davidbritch) [inline UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchXamlPage.xaml]: Our convention is to start names with a lower case char (as you've done in the code version).

File: UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchXamlPage.xaml [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+                   Text="The switch is off"
+                   VerticalOptions="CenterAndExpand" 
+                   HorizontalOptions="CenterAndExpand" />
+
+            <Switch x:Name="XAMLSwitch"
```
Comment (davidbritch) [inline UserInterface/SwitchDemos/SwitchDemos/SwitchDemos/SwitchXamlPage.xaml]: Our convention is to start names with a lower case char (as you've done in the code version).

File: UserInterface/SwitchDemos/SwitchDemos/SwitchDemos.Android/Properties/AndroidManifest.xml [hunk 8]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.companyname.SwitchDemos">
+    <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="28" />
```
Comment (davidbritch) [inline UserInterface/SwitchDemos/SwitchDemos/SwitchDemos.Android/Properties/AndroidManifest.xml]: Remove targetSdkVersion.

**Comment by davidbritch** [comment]
You also need to add screenshots to this sample. See the Slider project as an example.

**Pull request merged**